composer.json

```
@@ -33,7 +33,7 @@
     "php": "^7.1 || ^8.0",
     "ext-pdo": "*",
     "contao/core-bundle": "^4.4",
-    "doctrine/dbal": "^2.5",
+    "doctrine/dbal": "^2.5 || ^3.0",
```

Please increase requirements of version 2 to 2.11 as there the forward compatibility layer exists.

```suggestion
    "doctrine/dbal": "^2.11 || ^3.0",
```